roles/smallfile-bench/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Generate smallfile test input yaml
+  k8s:
+    definition:
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        namespace: '{{ operator_namespace }}'
+        name: smallfile-test
+      data:
+        smallfilejob: "{{ lookup('template', 'smallfilejob.yaml.j2') }}"
+  when: smallfile.clients > 0
+
+- name: Start Run(s)
+  k8s:
+    definition:
+      kind: Job
+      apiVersion: batch/v1
+      metadata:
+        name: '{{ meta.name }}-smallfile-client--benchmark'
+        namespace: '{{ operator_namespace }}'
+      spec:
+        ttlSecondsAfterFinished: 600
+        template:
+          metadata:
+            labels:
+              app: smallfile-benchmark
+          spec:
+            containers:
+            - name: benchmark-server
+              image: "quay.io/nichawla/smallfile_container:testing"
```

Comment:
We will want to build this into our benchmark-operator namespace

Reply:
I built one in [quay.io](https://quay.io/repository/bengland2/smallfile)  here using the aforementioned Dockerfile.  How does your image build differ from the one in the [smallfile tree here](https://quay.io/repository/bengland2/smallfile?tab=tags)?   I don't want to have too many of these images - for example if smallfile sources change, we need to be able to know which images to rebuild to distribute those changes.

Reply:
Ben, since I didn't have the idea about the available smallfile containers already , hence I constructed my own, but it can be replaced.